api/entity_config/config.go

```diff
@@ -12,6 +12,12 @@ func ConfigValue(network *net.Network, application, entity, config string) (stri
 	if nil != err {
 		return "", err
 	}
+
+	var value string;
+	err = json.Unmarshal(bytes, &value)
+	if nil == err {
```

Comment:
I personally prefer this to be the `nil != err` case, so that the last thing returned is the happy path, and maybe return the error too.